.circleci/config.yml

```diff
@@ -45,6 +45,31 @@ jobs:
       - restore_cache:
           key: vendor-{{ checksum "Gopkg.lock" }}
       - run: go build -o machine-controller github.com/kubermatic/machine-controller/cmd/controller
+      - save_cache:
+          key: machine-controller-{{ .Revision }}
+          paths:
+            - machine-controller
+
+  publish:
+    machine: true
+    steps:
```

Comment:
Specify:
`working_directory: /go/src/github.com/kubermatic/machine-controller`
On the publish level, so you dont have to do a change-directory

Reply:
Yeah I tried that, makes the build instantly fails because that dir doesn' exist

Reply:
Can you try again?
In https://github.com/kube-node/kube-machine/blob/master/.circleci/config.yml
We didn't had to do it